engine/runtime/src/main/java/org/enso/interpreter/runtime/data/vector/ArrayBuilder.java

```diff
@@ -243,7 +273,7 @@ public String toDisplayString(boolean ignore) {
     return "Array_Builder";
   }
 
-  Object asVector(boolean mustBeExact) {
+  Object asVector(boolean mustBeExact, boolean avoidWarningsIgnored) {
```

Comment:
ignored parameter?

Reply:
Yes, ignored parameter. Hence it has `...Ignored` suffix.